check_api/src/main/java/com/google/errorprone/JavacErrorDescriptionListener.java

```diff
@@ -106,6 +110,7 @@ public void onDescribed(Description description) {
           break;
       }
       log.report(factory.create(type, log.currentSource(), pos, MESSAGE_BUNDLE_KEY, message));
+      fileReporter.report(type, log.currentSource(), pos, message);
```

Comment:
I wonder whether we should have a composite reporter, to abstract the use of both of these `report` methods?  Taking a `JCDiagnostic` type as the param to `report` does not seem a terrible idea either.

Reply:
Though I haven't looked at where `Log` manifests from!